### PR TITLE
Rising Sun setup

### PR DESCRIPTION
**Pull request by sumpfork**
Starting a Rising Sun PR based on https://github.com/sumpfork/dominiontabs/issues/498#issuecomment-2369800260

Things I added:
- preliminary images
- test adjustments

### CODE REVIEW

**Review by spazard1 — APPROVED** [review]
LGTM.

File: src/domdiv/images/sun.png
Comment (nickv2002) [inline src/domdiv/images/sun.png:1]: I added this image which is just [referred to as a Sun or a Sun Token](https://wiki.dominionstrategy.com/index.php/Prophecy#Official_Rules).

File: src/domdiv/draw.py [hunk 1]
```diff
@@ -1193,6 +1193,8 @@ def replace_image_tag(
             # Potion
             (r"(\d+)\s*\<\*POTION\*\>", "potion_small.png", 2, 140, 1.5),
             (r"Potion", "potion_small.png", 1.2, 100),
+            # Sun
+            (r"Sun", "sun.png", 1.2, 105),
```
Comment (nickv2002) [inline src/domdiv/draw.py:1197]: I added the rule to replace it in card text (though I'm not sure if that would impact anywhere else yet). There are a few other cards that have "sun" in their text...

Comment (sumpfork) [reply src/domdiv/draw.py:1197]: I wonder whether it shouldn't be `SunToken` or `RisingSun` instead of `Sun` to avoid accidental use?

Comment (nickv2002) [reply src/domdiv/draw.py:1197]: I checked a few cards with "sun" in their names and this seems to work OK, so approving. Keep an eye on it though.

**Review by nickv2002 — APPROVED** [review]

File: src/domdiv/draw.py (hunk 1 above)
Comment (nickv2002) [reply src/domdiv/draw.py:1197]: I'm fine with changing it to `SunToken` Let me do that....

Comment (nickv2002) [reply src/domdiv/draw.py]: Done, changed to `SunToken`

**Comment by TistoH** [comment]
Thank you all for the work involved in integrating this into the main db and creating new images.  I only wish I could test it (my research says that a tool dependency prevents me from using cygwin as my development environment).

**Comment by nickv2002** [comment]
> Thank you all for the work involved in integrating this into the main db and creating new images. I only wish I could test it (my research says that a tool dependency prevents me from using cygwin as my development environment).

If you can use docker images you might try those. There's [some docs on the homepage](https://github.com/sumpfork/dominiontabs#docker) that work fine on this `rising_sun` branch to compile and output PDFs.

**Pull request merged**